android/src/main/assets/SearchWebView.js

```diff
@@ -0,0 +1,97 @@
+var MyApp_SearchResultCount = 0;
```

Comment:
What is MyApp?

Reply:
I migrate this code from the old code. So I followed that name.